src/applications/static-pages/facilities/vet-center/createExpandableOperatingStatus.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default async function createExpandableOperatingStatus() {
+  const statusWidgets = Array.from(
+    document.querySelectorAll(
+      'div[data-widget-type^=expandable-operating-status]',
+    ),
+  );
+
+  if (statusWidgets.length) {
+    const {
+      default: ExpandableOperatingStatus,
+    } = await import('../../shared/ExpandableOperatingStatus');
+
+    statusWidgets.forEach(el => {
+      if (
+        !el.attributes.status.nodeValue ||
+        el.attributes.status.nodeValue === 'normal'
+      )
+        return;
+      let status;
+      let iconType;
+
+      switch (el.attributes.status.nodeValue) {
+        case 'limited':
+          status = 'Limited services and hours';
+          iconType = 'triangle';
+          break;
+        case 'closed':
+          status = 'Facility closed';
+          iconType = 'circle';
+          break;
+        case 'notice':
+          status = 'Facility notice';
+          iconType = 'circle';
+          break;
+        default:
+          status = 'Facility status';
+          iconType = 'triangle';
+      }
```

Comment:
Maybe this logic should live in the `ExpandableOperatingStatus` component? (Easier to test, for one thing.)

Reply:
good idea, I was trying to make the component as generic as possible, I don't think other teams have no use it as operation status